gridData/core.py

```diff
@@ -363,10 +381,11 @@ def _export_python(self, filename):
         with open(filename, 'wb') as f:
             cPickle.dump(data, f, cPickle.HIGHEST_PROTOCOL)
 
-    def _export_dx(self, filename):
+    def _export_dx(self, filename, **kwargs):
```

Comment:
why not use `type=none` instead of undocumented `**kwargs`?

Reply:
changed